DB/NEW_KT_DB/DataAccess/ObjectManager.py

```diff
@@ -58,47 +80,52 @@ def convert_object_name_to_management_table_name(object_name):
         return f'mng_{object_name}s'
```

Comment:
if table_name is sent as a parameter to each function call, and db_manager.create_table uses that param and not this function call as the table name on creating, then this function is unnecessary.